### ISSUE TITLE
chore(pagination): pageChanged triggered multiple times

### ISSUE THREAD

**Issue by rhushi-lokhande** [valor-software/ngx-bootstrap#108]
I have posted my que 
http://stackoverflow.com/questions/35081261/ng2-bootstrap-pagination-pagechanged-triggered-multiple-times

**Comment by valorkin** [comment]
This is awesome! Now it is cristal clear where the problem
It could be solved with rxjs in one line!
This issues is actually a duplicate but has good info,
Thanks!

**Comment by bitbay** [comment]
same thing on

```
@Output() private numPages:EventEmitter<number> = new EventEmitter();
```

Wondering about the rxjs one-liner!

**Comment by valorkin** [comment]
actually in your code until it is fixed,
it could fixed with 
observe from event . distinct until changed
and that`s it

**Comment by valorkin** [comment]
should be fixed with #149
